### ISSUE TITLE
Found Devices: none

### ISSUE THREAD

**Issue by jayeshnaik** [mclarkk/lifxlan#53]
I am testing on Windows and not finding devices. verbose_lan shows packets found device none

**Comment by mclarkk** [comment]
Do you see them on your phone app?

**Comment by jayeshnaik** [comment]
Yes phone so showed items. 2 lamps

**Comment by jayeshnaik** [comment]
I have lifx color 1000 lamps

**Comment by mclarkk** [comment]
Do you have some kind of VPN setup? If not, is it possible that your network is blocking UDP on certain ports?

**Comment by aptitude** [comment]
I can confirm this issue on windows.

Windows with default settings probably interferes with the UDP traffic.

lifxlan works for me on the same computer with Linux as OS, but not with windows 10 or a Linux in a Virtual Machine hosted by Windows 10.

**Comment by mclarkk** [comment]
Figured it out! Windows doesn't handle the broadcast address "255.255.255.255" the same way as macOS and Linux. I was able to replicate the problem with a Windows 10 VM. No devices found using `hello_world.py`, even with the firewall completely off. After I replaced the global broadcast address with a broadcast address tailored to my local network address ("162.168.1.255"), it worked fine. I'm working on a more general solution now, and will hopefully get it working soon for those Windows users out there.
